packages/discord.js/src/managers/MessageManager.js

```diff
@@ -43,6 +43,8 @@ class MessageManager extends CachedManager {
 
   /**
    * Options used to fetch a message.
+   * <info>The `before`, `after`, and `around` parameters are mutually exclusive (on Discord API side),
```

Comment:
any sense of mentioning that it's exclusive on discord API side?

Reply:
This is on the wrong type definition. `FetchMessageOptions` is only for fetching a message. It should be on the one directly beneath this.



> any sense of mentioning that it's exclusive on _discord API side?_

That should be removed.

Reply:
you're right. I also messed up something on my side while commiting, so I'll close this and resubmit the PR